functions.js

```diff
@@ -9,7 +9,7 @@ import {
 
 let todos = [];
 
-const displayTask = () => {
+const display = () => {
   itemsDisplay.innerHTML = '';
   todos = JSON.parse(localStorage.getItem('storage-task')) || [];
```

Comment:
In the same way for display function you can use displayTask as it will make the function more specific.

Reply:
thank you!